scripts/ci-scripts/ci-eslint.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
```

Comment:
I didn't add this, and I'm not sure what it's purpose is.

Reply:
It's a "shebang".  It makes sure that `bash` is executing the script, no matter which shell inkoved it.

Reply:
I meant the intention of the script in general.